ddl/db_integration_test.go

```diff
@@ -357,6 +357,46 @@ func (s *testIntegrationSuite1) TestIssue4432(c *C) {
 	tk.MustExec("drop table tx")
 }
 
+func (s *testIntegrationSuite1) TestIssue5092(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+
+	tk.MustExec("create table t_issue_5092 (a int)")
+	tk.MustExec("alter table t_issue_5092 add column (b int, c int)")
+	tk.MustExec("alter table t_issue_5092 add column if not exists (b int, c int)")
+	tk.MustExec("alter table t_issue_5092 add column b1 int after b, add column c1 int after c")
+	// The following two statements are consistent with MariaDB.
+	tk.MustGetErrCode("alter table t_issue_5092 add column if not exists d int, add column d int", errno.ErrDupFieldName)
+	tk.MustExec("alter table t_issue_5092 add column d int, add column if not exists d int")
+	tk.MustExec("alter table t_issue_5092 add column if not exists (d int, e int), add column f int")
```

Comment:
Why do we support `if not exists` in TiDB? I try it in MySQL:
```
mysql> alter table t_issue_5092 add column if not exists d int;
ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'if not exists d int' at line 1
```

Reply:
`if not exists` is feature of MariaDB.  We support `if not exists` because we have support this feature before.

Reply:
Got it.